assets/js/googlesitekit/datastore/ui/constants.js

```diff
@@ -17,3 +17,5 @@
  */
 
 export const CORE_UI = 'core/ui';
+
+export const UI_VIEW_CONTEXT = 'viewContext';
```

Comment:
This is a bit confusing IMO as [`viewContext` is used elsewhere](https://github.com/google/site-kit-wp/blob/ba1ce47af7eccaaf46d465ab1361fc3cb424203d/assets/js/components/Root/index.js#L43-L53) as a Context value and prop. Can we rename this to be more specific to this? Maybe `ACTIVE_CONTEXT_ID` or `ACTIVE_SECTION_ID`? Given the use of the added `contextID` prop, maybe the former would be more accurate.

I'm referring to the name of the constant more than the value, but they should of course be consistent 😄

Reply:
Updated.